### PR TITLE
added expense tracker in vuejs

### PR DESCRIPTION
**Pull request by Maheshwari-Love**
Closes:  #572

- **Title** : Expense Tracker
- **Name:** :  Maheshwari Love
- **Idenitfy yourself:** : GSSOC

### Describe the add-ons or changes you've made 📃
Adde new project expense tracker in Vuejs 

### How Has This Been Tested? ⚙️

I have manually run the project on my local system

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:
added new project

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Maheshwari-Love** [comment]
@Avdhesh-Varshney please review this pr

**Comment by Maheshwari-Love** [comment]
@Avdhesh-Varshney   i have followed the guidlines. Can you please specify the requested changes

File: Vue-JS-Projects/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@
 |-------|--------------|----------|
 |   1   | [Markdown Previewer](./Basic/Markdown-Previewer) | ![Basic](https://img.shields.io/badge/Basic-00FF00?style=for-the-badge) |
 |   2   | [Quote-Generator](./Basic/Quote-Generator) | ![Basic](https://img.shields.io/badge/Basic-00FF00?style=for-the-badge) |
-
+|   3   | [Expense Tracker](./Intermediate/Expense-Tracker/) | ![Basic](https://img.shields.io/badge/Basic-00FF00?style=for-the-badge) |
```
Comment (Avdhesh-Varshney) [inline Vue-JS-Projects/README.md:30]: Attach the label of Intermediate

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Review by Avdhesh-Varshney — APPROVED** [review]

**Comment by Avdhesh-Varshney** [comment]
Properly link your issue with the pr. And also carefully attach the label in readme file.

@Maheshwari-Love Please fill the feedback form: https://forms.gle/Wvsp9N8TDF1WPStq9

**Comment by Maheshwari-Love** [comment]
> Properly link your issue with the pr. And also carefully attach the label in readme file.
> 
> @Maheshwari-Love Please fill the feedback form: https://forms.gle/Wvsp9N8TDF1WPStq9

okay i will take care of that

**Pull request merged**